Jenkinsfile

```diff
@@ -38,21 +36,26 @@ pipeline {
             stages{
                 stage('Checkout'){
                     steps {
-                        sh 'mkdir ${REPO}'
-                        // source checks require the directory
-                        // name to be the same as the repo name
                         dir("${REPO}") {
                             // checkout repo
                             checkout scm
                             installPipfile(false)
                             withVenv {
                                 withTools(params.TOOLS_VERSION) {
-                                    sh './tools/ci/checkout-submodules.sh'
+                                    dir("examples") {
+                                        sh 'cmake -B build -G "Unix Makefiles"'
+                                    }
                                 }
                             }
                         }
                     }
                 }
+                stage('Library checks') {
+                    steps {
+                        sh "tree"
```

Comment:
did you mean to leave it here?

Reply:
Nope - I'll remove it. It's my "where am I" debug method.